Backend/app/dashboard/CommonAPIs/src/Dashboard/RiderPlatform/Ride.hs

```diff
@@ -0,0 +1,83 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+{-# LANGUAGE DerivingVia #-}
+
+module Dashboard.RiderPlatform.Ride
+  ( module Dashboard.RiderPlatform.Ride,
+    module Reexport,
+  )
+where
+
+import Dashboard.Common as Reexport
+import qualified Dashboard.Common as DP
+import Kernel.External.Maps
+import Kernel.Prelude
+import Kernel.Types.Centesimal
+import Kernel.Types.Id
+import Servant
+
+---------------------------------------------------------
+-- share ride info--------------------------------------
+
+type ShareRideInfoAPI =
+  Capture "rideId" (Id DP.Ride)
+    :> "share"
+    :> Get '[JSON] ShareRideInfoRes
+
+data ShareRideInfoRes = ShareRideInfoRes -- handle repetition
```

Comment:
remove the comment

Reply:
Done

Reply:
@prashant601 I added this comment for you to handle repetitions of datatypes on dashboard side(RideStatus, BookingLocation, ShareRideInfo etc.) if required.